docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -63,17 +61,18 @@ The preceding code demonstrates the following `Azure.Core` convenience method pa
 
 The following code uses a `ContentSafetyClient` to call the `AnalyzeText` protocol method:
 
-:::code source="snippets/protocol-convenience-methods/AzureCoreProtocol/Program.cs" highlight="19-24":::
+:::code source="snippets/protocol-convenience-methods/AzureCore/Protocol/Program.cs" highlight="19-24":::
 
-The preceding code demonstrates the following protocol method patterns:
+The preceding code demonstrates the following `Azure.Core` protocol method patterns:
 
-- Uses the `RequestContent` type to supply data for the request body.
-- Uses the `RequestContext` type to configure request options.
-- Returns data using the `Response` type.
-- Reads content from the response data into a [dynamic](../../csharp/advanced-topics/interop/using-type-dynamic.md) type using <xref:Azure.AzureCoreExtensions.ToDynamicFromJson%2A>. For more information, see [Announcing dynamic JSON in the Azure Core library for .NET](https://devblogs.microsoft.com/azure-sdk/dynamic-json-in-azure-core/).
+1. Create the request, using a `RequestContent` object for the request body.
```

Comment:
Yeah, I love this.  One thing I think this serves to do is show that the usage pattern for both Azure.Core and SCM-based protocol methods is essentially the same, just with different type names.  If it made sense, it might even be worth considering highlighting this point closer to the intro.  Another possible thing to consider if it seems like a good idea to you, is to e.g. bold the steps in both Azure and SCM samples to make them pop a bit more, e.g.:

1. **Create the request**, using a `RequestContent` ...

But I defer to you on the value of those suggestions in the context of what you're targeting for this audience.

Reply:
Great idea! I'll incorporate that suggestion.